Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -231,6 +231,12 @@ public Vector3 SliderTrackDirection
         #endregion  
 
         #region Unity methods
+
+        private void OnEnable()
+        {
+            // Register for bubble up events
+            CoreServices.InputSystem?.RegisterPropagationHandler<IMixedRealityPointerHandler>(this);
```

Comment:
We have several other components that handle pointer events, and in particular I see people use PointerHandler quite often. What are your thoughts on either modifying PointerHandler to support event propagation, or creating a a new component such as PointerPropagationHandler (or similar) to allow devs to specify the event propagation phase to subscribe to?

Reply:
My thinking is that all handlers should handle bubble up by default without any code change, and only use registration if tickle down is needed. But maybe this can be done on my next PR for the feature branch?